src/odemis/gui/cont/tabs.py

```diff
@@ -1543,9 +1522,14 @@ def display_new_data(self, filename, data):
                    self.panel.vp_inspection_br]:
             vp.canvas.fit_view_to_next_image = True
 
+        if isinstance(data[0], model.DataArray):
```

Comment:
No need to differentiate: it's either a list of DA or DAS, but in both cases, the metadata is accessed the same way.